packages/rollup-plugin/test/rollup-plugin.test.ts

```diff
@@ -0,0 +1,59 @@
+import { rollup, OutputOptions } from 'rollup';
+import esbuild from 'rollup-plugin-esbuild';
+import json from '@rollup/plugin-json';
+import path from 'path';
+
+import { vanillaExtractPlugin } from '..';
+
+async function buildAndMatchSnapshot(outputOptions: OutputOptions) {
```

Comment:
Rather than create a whole new fixture, I moved the tests to live with the plugin itself and use the existing themed fixture.